src/Statues/slavers.js

```diff
@@ -0,0 +1,10 @@
+const SlaverStatues = [ 
```

Comment:
I think this should be named `slaverStatueList`.

Reply:
done